nilearn/interfaces/bids/glm.py

```diff
@@ -374,10 +403,10 @@ def save_glm_to_bids(
         }
 
         for map_name, img in renamed_contrast_maps.items():
-            out_file = os.path.join(out_dir, map_name)
-            img.to_filename(out_file)
+            img.to_filename(out_dir / map_name)
 
     # Write out model-level statistical maps
+    # TODO: only one resiudal / rsquare for all the runs?
```

Comment:
From here to line 418

Maybe I am missing something, but we should get one Rsquared and one error image per run, no?

@tsalo

Reply:
I think you're right. Based on `FirstLevelModel._get_voxelwise_model_attribute` it looks like r-squared is calculated for each design matrix in the model.

Reply:
I don't see why there should be only one for all runs.

Reply:
OK will fix this

Reply:
fixed